actions/vulnerability_scan/action.yaml

```diff
@@ -0,0 +1,50 @@
+name: Vulnerability Scanning

```

Comment:
This action will be used for the other two shared action, but do that as a part of UID2-2681.